golden_response/staging/get_stat_all/result.json

```diff
@@ -1,715 +1,765 @@
+        "2013":  38260787,
+        "2014":  38596972,
+        "2015":  38918045,
+        "2016":  39167117,
+        "2017":  39358497,
+        "2018":  39461588,
+        "2019":  39512223
        },
-       "measurementMethod": "CensusPEPSurvey",
-       "importName": "CensusPEP",
-       "provenanceDomain": "census.gov"
+       "measurementMethod":  "CensusPEPSurvey",
+       "importName":  "CensusPEP",
+       "provenanceDomain":  "census.gov"
       },
       {
-       "val": {
-        "2011": 36969200,
-        "2012": 37325068,
-        "2013": 37659181,
-        "2014": 38066920,
-        "2015": 38421464,
-        "2016": 38654206,
-        "2017": 38982847,
-        "2018": 39148760
+       "val":  {
+        "2011":  36969200,
+        "2012":  37325068,
+        "2013":  37659181,
+        "2014":  38066920,
+        "2015":  38421464,
+        "2016":  38654206,
+        "2017":  38982847,
+        "2018":  39148760
        },
-       "measurementMethod": "CensusACS5yrSurvey",
-       "importName": "CensusACS5YearSurvey",
-       "provenanceDomain": "census.gov"
+       "measurementMethod":  "CensusACS5yrSurvey",
+       "importName":  "CensusACS5YearSurvey",
+       "provenanceDomain":  "census.gov"
+      },
+      {
+       "val":  {
+        "2010":  37349363,
+        "2011":  37691912,
+        "2012":  38041430,
+        "2013":  38332521,
+        "2014":  38802500,
+        "2015":  39144818,
+        "2016":  39250017,
+        "2017":  39536653,
+        "2018":  39557045,
+        "2019":  39512223
+       },
+       "measurementMethod":  "CensusACS1yrSurvey",
+       "importName":  "CensusACS1YearSurvey",
+       "provenanceDomain":  "census.gov"
       }
      ]
     }
    }
   },
-  "geoId/0649670": {
-   "statVarData": {
-    "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita": {},
-    "Count_CriminalActivities_CombinedCrime": {
-     "sourceSeries": [
+  "geoId/0649670":  {
+   "statVarData":  {
+    "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita":  {},
+    "Count_CriminalActivities_CombinedCrime":  {
+     "sourceSeries":  [
       {
-       "val": {
-        "2008": 2116,
-        "2009": 2155,
-        "2010": 1633,
-        "2011": 1509,
-        "2012": 1581,
-        "2013": 1867,
-        "2014": 1770,
-        "2015": 2201,
-        "2016": 1913,
-        "2017": 2138
+       "val":  {
+        "2008":  2116,
+        "2009":  2155,
+        "2010":  1633,
+        "2011":  1509,
+        "2012":  1581,
+        "2013":  1867,
+        "2014":  1770,
+        "2015":  2201,
+        "2016":  1913,
+        "2017":  2138
        },
-       "observationPeriod": "P1Y",
-       "importName": "FBIGovCrime",
-       "provenanceDomain": "fbi.gov"
+       "observationPeriod":  "P1Y",
+       "importName":  "FBIGovCrime",
+       "provenanceDomain":  "fbi.gov"
       }
      ]
     },
-    "Count_Person": {
-     "sourceSeries": [
+    "Count_Person":  {
```

Comment:
Git isn't rendering this Count_Person section well, but here the diffs are just adding CensusACS1yrSurvey